CHANGELOG.md

```diff
@@ -1,5 +1,200 @@
 # Changelog
 
+## [1.0.0-Release Candidate 1](https://github.com/ably/ably-asset-tracking-swift/tree/v1.0.0-rc.1)
+[Full Changelog](https://github.com/ably/ably-asset-tracking-swift/compare/v1.0.0-beta.1...v1.0.0-rc.1)
```

Comment:
To stay consistent with previous changelog entries, please insert a line of vertical whitespace above this line, separating it from the title. Some readers may consume this document in raw format, not rendered.

Reply:
Fixed in [e2cbd82](https://github.com/ably/ably-asset-tracking-swift/pull/322/commits/e2cbd82ad81e7780ba1e00d6cf4f5e581768340d)